server/internal/usecase/interfaces/item.go

```diff
@@ -99,4 +102,12 @@ type Item interface {
 	Publish(context.Context, id.ItemIDList, *usecase.Operator) (item.VersionedList, error)
 	Unpublish(context.Context, id.ItemIDList, *usecase.Operator) (item.VersionedList, error)
 	Import(context.Context, ImportItemsParam, *usecase.Operator) (ImportItemsResponse, error)
+	// ItemsAsCSV exports items data in content to csv file by modelID.
+	ItemsAsCSV(context.Context, id.ModelID, *int, *int, *usecase.Operator) (*io.PipeReader, error)
+	// ItemsAsGeoJSON converts items to Geo JSON type given a model ID
+	ItemsAsGeoJSON(context.Context, id.ModelID, *int, *int, *usecase.Operator) (*integrationapi.FeatureCollection, error)
+	// ItemsWithProjectAsCSV converts items content to CSV given by project ID or project Alias and model ID or model Key
+	ItemsWithProjectAsCSV(context.Context, project.IDOrAlias, model.IDOrKey, *int, *int, *usecase.Operator) (*io.PipeReader, error)
+	// ItemsWithProjectAsGeoJSON converts items content to Geo JSON given by project ID or project Alias and model ID or model Key
+	ItemsWithProjectAsGeoJSON(context.Context, project.IDOrAlias, model.IDOrKey, *int, *int, *usecase.Operator) (*integrationapi.FeatureCollection, error)
```

Comment:
```suggestion
	// ItemsAsCSV exports items data in content to csv file by modelID.
	ItemsAsCSV(context.Context, id.ModelID, *int, *int, *usecase.Operator) (*io.PipeReader, error)
	// ItemsAsGeoJSON converts items to Geo JSON type given a model ID
	ItemsAsGeoJSON(context.Context, id.ModelID, *int, *int, *usecase.Operator) (*integrationapi.FeatureCollection, error)
```
No need for 2 implementations, you can get the model id in adapter level and call the same implementation